packages/fast-components-react-base/src/tabs/tab.tsx

```diff
@@ -59,3 +59,4 @@ class Tab extends Foundation<TabHandledProps, TabUnhandledProps, {}> {
 
 export default Tab;
 export * from "./tab.props";
+export { TabClassNameContract };
```

Comment:
same issue here, we have only been exporting from base components the handledProps and unhandledProps